flint/flint-spark-integration/src/main/scala/org/opensearch/flint/spark/skipping/partition/PartitionSkippingStrategy.scala

```diff
@@ -7,14 +7,24 @@ package org.opensearch.flint.spark.skipping.partition
 
 import org.opensearch.flint.spark.skipping.FlintSparkSkippingStrategy
 
+import org.apache.spark.sql.Column
+import org.apache.spark.sql.catalyst.expressions.aggregate.{AggregateFunction, First}
+
 /**
  * Skipping strategy for partitioned columns of source table.
  */
-class PartitionSkippingStrategy(override val indexedColumn: (String, String))
+class PartitionSkippingStrategy(
+    override val kind: String = "partition",
+    override val columnName: String,
+    override val columnType: String)
     extends FlintSparkSkippingStrategy {
 
   override def outputSchema(): Map[String, String] = {
-    Map(indexedColumn._1 -> convertToFlintType(indexedColumn._2))
+    Map(columnName -> convertToFlintType(columnType))
```

Comment:
does the flintType align need to align with AggregateFunction output type?

Reply:
yes, it needs to align. As comment above, I may need to use AggregateFunction output type if `columnType` is unnecessary. I've noted this down. Thanks!